docs/docs/100-reference/01-command-line/acorn_install.md

```diff
@@ -27,6 +27,8 @@ acorn install
       --cluster-domain strings                The externally addressable cluster domain (default .on-acorn.io)
       --controller-replicas int               acorn-controller deployment replica count
       --default-publish-mode string           If no publish mode is set default to this value (default user)
+      --default-service-memory int            
```

Comment:
service isnt the word we use. we use container.

Reply:
Hm. good point. i still dont love using that word. i'd say `--default-memory` rather than service. Workload is another possibility. let's touch on in standup.

Reply:
For those coming to read this later, we decided to reference the Acorn concept of Jobs and Containers as workloads.